test/aspects/ec2-automated-shutdown/handler/index.test.ts

```diff
@@ -0,0 +1,222 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import { EC2Client } from '@aws-sdk/client-ec2';
+import { Context } from 'aws-lambda';
+import { handler } from '../../../../src/aspects/ec2-automated-shutdown/handler';
+
+jest.mock('@aws-sdk/client-ec2');
+
+describe('Lambda Handler', () => {
+    let mockContext: Context;
+    let mockEvent: any;
+    const mockEC2Client = {
+        send: jest.fn()
+    };
+
+    beforeEach(() => {
+        mockContext = {
+            awsRequestId: 'test-request-id'
+        } as Context;
+
+        mockEvent = {
```

Comment:
Can we store these mocks in a separate file? In other places we put them in a "fixtures" folder under the test construct folder. Check out the friendly embrace construct for an example